_studio/mfx_lib/shared/include/mfx_interface_scheduler.h

```diff
@@ -88,6 +88,10 @@ class MFXIScheduler : public MFXIUnknown
     virtual
     mfxStatus AddTask(const MFX_TASK &task, mfxSyncPoint *pSyncPoint) = 0;
 
+    // Find the oldest active syncPoint for a specified owner
+    virtual
+    mfxStatus FindOldestActiveSyncPoint(const void *pOwner, mfxSyncPoint *pSyncPoint) = 0;
```

Comment:
I hate myself to say that, but we can't add it here. I afraid `MFXIScheduler3` is coming...